### PR TITLE
Change type for $wgDefaultSkin to "skins"

### PR DESCRIPTION
**Pull request by Universal-Omega**
This comes after I added the skins type to ManageWiki (miraheze:managewiki[#192](https://github.com/miraheze/ManageWiki/pull/192)).

Should not be merged until @JohnFLewis finishes what is doing in CreateWiki/ManageWiki and updates ManageWiki in miraheze:mediawiki!

### CODE REVIEW

File: ManageWikiSettings.php [hunk 1]
```diff
@@ -7,7 +7,7 @@
  *
  * check: adds a checkbox.
  * integer: adds a textbox with integer validation (requires: minint and maxint which are minimum and maximum integer values).
- * language: adds a dropdown for language selection (all which are known to MediaWiki).
+ * language: adds a dropdown for language selection (all which are known to MediaWiki).f
```
Comment (RhinosF1) [inline ManageWikiSettings.php:10]: Why the random trailing f?

File: LocalSettings.php [hunk 2]
```diff
@@ -3241,8 +3241,7 @@
 			'monobook',
 			'vector',
 			'timeless',
-			'minerva',
-			'minervaneue'
```
Comment (RhinosF1) [inline LocalSettings.php:3245]: Did you intend to remove a skin here?

**Review by RhinosF1 — CHANGES_REQUESTED** [review]
-

File: LocalSettings.php (hunk 2 above)
Comment (Universal-Omega) [reply LocalSettings.php:3245]: Yes, keeping minerva, removing minervaneue, as minervaneue does nothing

File: ManageWikiSettings.php (hunk 1 above)
Comment (Universal-Omega) [reply ManageWikiSettings.php:10]: Oh, whoops. Let me fix now.

Comment (Universal-Omega) [reply ManageWikiSettings.php]: fixed

**Pull request merged**